queries/java/highlights.scm

```diff
@@ -35,14 +35,15 @@
 ; Annotations
 
 (annotation
-  name: (identifier) @attribute)
+  "@" @annotation
+  name: (identifier) @annotation.name)
 (marker_annotation
-  name: (identifier) @attribute)
+  "@" @annotation
+  name: (identifier) @annotation.name)
```

Comment:
All of these should be `@attribute`, the CI doesn't accept annotation as a valid name

Reply:
I don't understand. @amaanq asked it to be @annotation or am I missing something here?

Reply:
whoops, mixed up annotation and attribute, but yeah `@attribute` is fine

Reply:
I changed it. Please review again